backend/app/main.py

```
@@ -62,18 +68,60 @@
+            len(orphan_ids),
+        )
+        db.execute(delete(DatasetRow).where(DatasetRow.dataset_id.in_(orphan_ids)))
+        db.execute(delete(DatasetColumn).where(DatasetColumn.dataset_id.in_(orphan_ids)))
+        db.execute(delete(Dataset).where(Dataset.id.in_(orphan_ids)))
+        db.commit()
+
+    for did in orphan_ids:
+        try:
+            clear_index_job(int(did))
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)